From b476ff908f65e4825435d6fee45251c4f2a168fa Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 13 Aug 2020 11:57:10 -0700
Subject: [PATCH 1519/1852] v4l: xilinx: vipp: remove unused variable

commit 09a75e88c95e587bcb262665d688df98d646f36a from
https://github.com/Xilinx/linux-xlnx.git

drivers/media/platform/xilinx/xilinx-vipp.c: In function 'xvip_graph_entity_start_stop':
drivers/media/platform/xilinx/xilinx-vipp.c:305:22: warning: variable 'subdev' set but not used [-Wunused-but-set-variable]
  305 |  struct v4l2_subdev *subdev;
      |                      ^~~~~~

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vipp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vipp.c b/drivers/media/platform/xilinx/xilinx-vipp.c
index 0473d98359bf..4e092ea0f62c 100644
--- a/drivers/media/platform/xilinx/xilinx-vipp.c
+++ b/drivers/media/platform/xilinx/xilinx-vipp.c
@@ -302,7 +302,6 @@ static bool xvip_graph_entity_start_stop(struct xvip_composite_device *xdev,
 {
 	unsigned long pad_flag = on ? MEDIA_PAD_FL_SOURCE : MEDIA_PAD_FL_SINK;
 	unsigned int i;
-	struct v4l2_subdev *subdev;
 	bool state;
 	int ret;
 
@@ -338,7 +337,6 @@ static bool xvip_graph_entity_start_stop(struct xvip_composite_device *xdev,
 	}
 
 	/* set state and report if state is changed or not */
-	subdev = media_entity_to_v4l2_subdev(entity->entity);
 	state = xvip_graph_entity_set_streaming(xdev, entity, on);
 	/* This shouldn't happen as check is already above */
 	if (state == on) {
-- 
2.31.1

